frontend/lib/src/components/elements/GraphVizChart/GraphVizChart.tsx

```diff
@@ -38,75 +38,68 @@ interface Dimensions {
 const dummyGraphviz = graphviz
 dummyGraphviz // eslint-disable-line @typescript-eslint/no-unused-expressions
 
+const isFullScreen = (height = 0): boolean => Boolean(height)
+
 export function GraphVizChart({
   width: propWidth,
   element,
   height: propHeight,
 }: GraphVizChartProps): ReactElement {
   const chartId = `graphviz-chart-${element.elementId}`
 
+  const isFull = isFullScreen(propHeight)
```

Comment:
Think this could just be simplified to `Boolean(propHeight)`

Reply:
Agreed! Fixed 😄